monai/data/dataset.py

```diff
@@ -773,6 +776,8 @@ def __init__(
                 the dataset has duplicated items or augmented dataset.
             hash_func: if `hash_as_key`, a callable to compute hash from data items to be cached.
                 defaults to `monai.data.utils.pickle_hashing`.
+            runtime_cache: whether to compute cache at the runtime, default to `False` to prepare
+                the cache content at initializaiton.
```

Comment:
may need some docstring about the dataloader's `num_workers`, if `num_workers==0`, this `set_multiprocessing_cache(False)` is needed..

Reply:
Hi @wyli , if `num_workers==0`, no need to call `set_multiprocessing_cache(False)`. It's only necessary when caching on GPU memory, and the `monai.data.DataLoader` will seamlessly take care of it in this PR.

Thanks.

Reply:
added doc-string in the latest commit.

Thanks.